### DISCUSSION TITLE
提过一个免费获取通过 DeepL API 地址验证的方法

### DISCUSSION THREAD

**Discussion by MSDNQ** [hcfyapp/crx-selection-translate#1371] [Polls]
~~注：卡在地址验证的，请详细填写真实地址，我在文章中已经提供图片模板了。~~
~~https://ani365.notion.site/2022-5-3-DeepL-API-0348b1b42c2e45ee8bf116445cb72545~~
5.3日已失效，[bankoff关闭服务](https://ani365.notion.site/2022-5-3-DeepL-API-0348b1b42c2e45ee8bf116445cb72545)，有其他免费渠道开虚拟v/m卡的可以下方留言。
~~具体操作步骤已写在notion中，https://ani365.notion.site/DeepL-API-0348b1b42c2e45ee8bf116445cb72545~~

**Comment by cotagxrdhy** [comment]
感谢，但账号估计很快会被封掉吧

**Comment by MSDNQ** [comment]
> 感谢，但账号估计很快会被封掉吧

这个方法就是获取一张visa信用卡，deepl目前这方面验证不严格，0余额的也是可以过的。之前开的测试用的账户用的也是虚拟v/m卡过的，用了一年多也没事。

**Reply by poiu23** [reply]
> > 感谢，但账号估计很快会被封掉吧
> 
> 这个方法就是获取一张visa信用卡，deepl目前这方面验证不严格，0余额的也是可以过的。之前开的测试用的账户用的也是虚拟v/m卡过的，用了一年多也没事。

目前好像已经失效了，按教程步骤最后会提示出错。

**Reply by MSDNQ** [reply]
是的已经失效了，bankoff停止服务了。之前注册的如果已经通过deepl验证的仍然可用。

**Comment by younyokel** [comment]
Your card number is incorrect. (Error Code: E6GMAL)

**Comment by shangdev** [comment]


**Reply by EricWong1994** [reply]
你开的高级卡还是白金卡？这网站长久吗

**Reply by shangdev** [reply]
开的最基础的
